solutions/usaco-1011.mdx

```diff
@@ -95,4 +95,58 @@ public class triangles{
 
 </JavaSection>
 
+<CPPSection>
+
+```CPP
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	freopen("triangles.in", "r", stdin);
+	freopen("triangles.out", "w", stdout);
+	int n; cin >> n;
+	int x[n];
```

Comment:
https://usaco.guide/bronze/intro-ds?lang=cpp#dynamic-arrays

Reply:
Do I really need to use dynamic arrays for this problem? Regular arrays in bronze should be good enough right? I don't think people in bronze know what dynamic arrays are to be honest.

Reply:
> Do I really need to use dynamic arrays for this problem? 

as long as you don't declare an array of variable length

> I don't think people in bronze know what dynamic arrays are to be honest.

But this module is in the Bronze section ...

Reply:
ok if I just make it global with size maxN then it should be fine right?

Reply:
or like maxN + 5 or something

Reply:
not sure why you would prefer to do that over a vector, but okay

Reply:
ok fixed 
eh i just don't like to use vectors for problems I can use arrays in I just personally feel that arrays are easier to work with 
(dont strike me down mighty benq pls)